### ISSUE TITLE
409: Label name exists or conflicts - aborted

### ISSUE THREAD

**Issue by spike4848** [jay0lee/got-your-back#105]
Hi,

I've been having problems restoring .mbox files to both Gmail accounts and also Google for Apps accounts. They were previously exported via Google Take out, and many emails had labels applied to them. In each instance, the restore gets to a certain message, (in this case 436 out of 3505) and I get the error -

409: Label name exists or conflicts - aborted

and it can't move past it. I've tried using --fast-incremental as an additional command and also purging the labels before restarting the restore, but it doesn't work.

Any help would be hugely appreciated!

Confirming that I have upgraded to the latest GYB release, and I am typing the commands as described in the GAM Wiki.

**Comment by markleinhos** [comment]
I got the same message. I renamed the label in the destination mailbox (LEGAL changed to LEGAL2) and the restore was able to continue. Will report back if it bombs again (39,000 messages to go!).

**Comment by JeDay17** [comment]
I am running into this same problem repeatedly with the same email address being restored to an archive address.  I begin the process and a few emails in it says  "label name exists or conflicts" even if it's a new label.  This is happening for two totally different email addresses being uploaded to the same archive address

**Comment by zyaldo** [comment]
I got the issue 400: Invalid label name - invalidArgument while restoring label with french caracters (I think base on the debug log) like é, î and my be a slash "/"  into the label could had lead to this error. 

I tried to rename de destination label but the next mail tag with the same label create the label again and same error. The --stipe-label solve the problem for me as I didn't need to keep label on destination

I'm not that a super user with gyb so if I could had played with the internal DB to stripe this particular label then I would had use that option instead of killing all label on all my e-mails.
